### ISSUE TITLE
Long content in tables should be viewable on hover (alt text?) 

### ISSUE THREAD

**Issue by aih** [aih/FlatGov#66]
The titles, and possibly lists of sponsors may not all show in the columns of tables. If it is easy (i.e. built in to the table library), add the ability to view the cell content on hover. This is also important for accessibility.

**Comment by aih** [comment]
![image](https://user-images.githubusercontent.com/217356/100378305-b36a0800-2fc7-11eb-9cf8-da3050f8c342.png)

**Comment by kapphire** [comment]
Added title to show the whole text.

**Comment by aih** [comment]
Working in  b786f

We may want to consider adding a clickable tooltip for this content (to click the truncated content and have the full content show up in a tooltip). Closing this issue.
